server.js

```diff
@@ -37,7 +37,8 @@ app.use(express.static('public'))
 app.post(['/workflows/*', '/triggers/*'], (req, res) => {
     res.send('ok');
     const payload = req.body;
-    const flatPayload = flatten(payload);
+    // Slack restricts variable keys to "a combination of letters, numbers, hyphens, and underscores."
+    const flatPayload = flatten(payload, { delimiter: '_' });
```

Comment:
@ChrisCarini, what about this idea to avoid needing to append the `?delimiter=_` URL param?

```suggestion
     // Slack's new Workflow builder restricts variable keys to
    // "a combination of letters, numbers, hyphens, and underscores."
    //
    // Maintain backwards compatibility with legacy workflow builder
    // where a '.' is supported to delimit flattened payloads
    const delimiter = req.path.includes('/triggers/') ? '_' : '.';
    const flatPayload = flatten(payload, { delimiter: delimiter });
```

Reply:
@amarinelli - sure! I like that (avoiding adding a URL param) even better than my suggestion!

I'll admit, I had to search for what the difference between the two paths (`/triggers/` vs `/workflows/`) were, and the best I could find was https://github.com/stevengill/pancake-studio/issues/4 that you happened to open - looks like `/triggers/` is the path for the new workflows, and given that's where most people will be creating workflows (especially after 2024-09-12, when legacy workflows become inaccessible), I think this makes sense instead of my suggestion!

I'd just be happy to see something merged in so I can use this! _(Also, cool to see a handful of Slack employees spun up this service for folks - of course I know that does not mean it's 'officially' supported by Slack, but neat none the less! 😄 )_

@mwbrooks / @cookbr / @stevengill - would love to have this reviewed, merged in, and deployed for broad use!